static/app/components/replays/replayController.tsx

```diff
@@ -118,6 +118,14 @@ const IconClockwise = styled(IconRefresh)`
   transform: scaleX(-1);
 `;
 
+const StyledCompactSelect = styled(CompactSelect)`
+  ${DropdownTriggerButton} {
+    padding: 0 ${space(1)};
+    height: 1.75rem;
+    min-height: 0;
+  }
```

Comment:
Oh you know what, can you not use the button `size` prop here? cc @vuluongj20 

I'm not certain we need to custom style it

Reply:
Are you meaning like passing `size` into the button style from in here?

Reply:
I'm suggesting like, maybe we don't need to restyle the button, can we just use the `size="zero"` prop on `StyledCompactSelect`?

Reply:
Doesn't seem to work, the button goes back to its regular height. 
<img width="285" alt="image" src="https://user-images.githubusercontent.com/7014871/167206691-54891564-ae40-488c-b294-cc507645a619.png">

Reply:
cc @vuluongj20 

you definitely shouldn't need to override the button style like this

Reply:
you can pass `triggerProps={{size: 'zero'}}` to `<CompactSelect>`

Reply:
Ahh got it, updated.